unblob/cli.py

```diff
@@ -23,16 +25,46 @@ def show_external_dependencies(
     if not value or ctx.resilient_parsing:
         return
 
-    dependencies = get_dependencies(ALL_HANDLERS)
+    plugin_manager = ctx.params["plugin_manager"]
+    handlers = ctx.params["handlers"]
+    plugins = ctx.params.get(
+        "plugins_path"
+    )  # may not exist, depends on parameter order...
+    handlers = add_handlers_from_extensions(plugin_manager, handlers, plugins)
+
+    dependencies = get_dependencies(handlers)
     text = pretty_format_dependencies(dependencies)
     exit_code = 0 if all(dep.is_installed for dep in dependencies) else 1
 
     click.echo(text)
     ctx.exit(code=exit_code)
 
 
+def add_handlers_from_extensions(
+    plugin_manager: UnblobPluginManager,
+    handlers: Handlers,
+    plugin_path: Optional[Path],
+):
+    if plugin_path:
+        plugin_manager.import_path(plugin_path)
+
+    plugin_manager.load_setuptools_entrypoints("unblob_v1")
+
+    plugins = [name for name, _plugin in plugin_manager.list_name_plugin()]
+    if not plugins:
+        return handlers
+
+    logger.info("Loaded plugins", plugins=plugins)
+
+    extra_handlers = list(itertools.chain(*plugin_manager.hook.unblob_register_handlers()))  # type: ignore
+
+    logger.debug("Loaded handlers from plugins", handlers=extra_handlers)
+
+    return handlers.with_prepended(extra_handlers)
```

Comment:
do we always want to add extra handlers at first prio.
This can be useful for overriding builtin ones, but still wondering. Probably wont hurt.

Reply:
That seemed like a good default. A possible way to make it more flexible is to pass the Handlers object to the hook callback instead of requiring it to return a list of handlers. That way it would be forward-compatible with any future API extensions.